src/components/ProjectField/TextAreaDisplay.tsx

```diff
@@ -21,9 +22,19 @@ const ProjectFieldTextAreaDisplay = ({ label, value }: ProjectFieldProps) => {
         <Typography fontSize={'16px'} lineHeight={'24px'} fontWeight={600} marginBottom={theme.spacing(1)}>
           {label}
         </Typography>
-        <Typography fontSize={'16px'} lineHeight={'24px'} fontWeight={400}>
-          {value}
-        </Typography>
+        {typeof value === 'string' && (
+          <Typography fontSize={'16px'} lineHeight={'24px'} fontWeight={400}>
```

Comment:
I think instead of using a TextField, you can just add `whiteSpace={'pre-wrap'}` here.

Reply:
Oh, nice! Didn't know that existed. It does the trick.